static/app/views/dashboards/widgetBuilder/components/newWidgetBuilderSlideout.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import SlideOverPanel from 'sentry/components/slideOverPanel';
+import {IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+type WidgetBuilderSlideoutProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export default WidgetBuilderSlideout;
+
+function WidgetBuilderSlideout({isOpen, onClose}: WidgetBuilderSlideoutProps) {
+  return (
+    <SlideOverPanel collapsed={!isOpen} slidePosition="left">
+      <SlideoutHeaderWrapper>
+        <SlideoutTitle>{t('Create Custom Widget')}</SlideoutTitle>
+        <CloseButton
+          priority="link"
+          size="zero"
+          borderless
+          aria-label={t('Close Widget Builder')}
+          icon={<IconClose size="sm" />}
+          onClick={onClose}
+        >
+          {t('Close')}
+        </CloseButton>
+      </SlideoutHeaderWrapper>
+    </SlideOverPanel>
+  );
+}
+
+const CloseButton = styled(Button)`
+  color: ${p => p.theme.gray300};
+  height: fit-content;
+  &:hover {
+    color: ${p => p.theme.gray400};
+  }
+  z-index: 100;
```

Comment:
I was able to take this out and it worked fine, is it necessary?

Reply:
whoops no not really. This was a copy-paste mistake lolz